tests/unit/dns_test.cc

```diff
@@ -63,15 +63,20 @@ static future<> test_bad_name(dns_resolver::options opts) {
     });
 }
 
-SEASTAR_TEST_CASE(test_resolve_udp) {
+using enable_if_with_networking = boost::unit_test::enable_if<SEASTAR_TESTING_WITH_NETWORKING>;
+
+SEASTAR_TEST_CASE(test_resolve_udp,
```

Comment:
Before the change this test (in fact, all the tests in dns_test.cc) ran regardless of SEASTAR_TESTING_WITH_NETWORKING, not they are conditionally skipped. These are tests that are described by

> several other internet-dependent tests were not properly disabled, causing test failures in environments without internet access.

, right?

Reply:
@xemul correct.